### PR TITLE
Implement Persistent Datastore w/ Handler in /data POST

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR is to be merged in conjunction with #28 to complete handling the comment section POST functionality.

- **Add datastore dependecy**
- **Create/Update Datastore from POST method**. The current implementation uses hard coded values, which will be changed after #28 is successfully merged.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -29,4 +32,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println("Hello Sam");
   }
+
+  public void updateDatastore(String nickname, String comment) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("nickname",nickname);
```
Comment (krasimirgg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: add a space after ','

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -29,4 +32,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    commentEntity.setProperty("comment",comment);
```
Comment (krasimirgg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: add a space after ','

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -29,4 +32,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    datastore.put(commentEntity);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    updateDatastore("placeholder_nickname","placeholder_comment");
```
Comment (krasimirgg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: add a space after ','

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -29,4 +32,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println("Hello Sam");
   }
+
+  public void updateDatastore(String nickname, String comment) {
```
Comment (krasimirgg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: this can be `private`

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (samuelsogbesan) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Updated with commit 4cb7874

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Updated with commit 9617098

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Updated with commit 9617098

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Updated with commit 9617098

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (krasimirgg) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: there's still a space missing between `"placeholder_nickname",` and `"placeholder_comment"` on line 46

**Review by krasimirgg — CHANGES_REQUESTED** [review]

**Comment by samuelsogbesan** [comment]
Updated in commit 24a8a97

**Review by krasimirgg — APPROVED** [review]
Good!

**Pull request merged**